Dockerfile

```diff
@@ -21,7 +21,22 @@ RUN TF_VERSION="0.11.3"; \
     unzip terraform_${TF_VERSION}_linux_amd64.zip -d /bin && \
     rm -f terraform_${TF_VERSION}_linux_amd64.zip
 
-# Get ct_config terraform provider.
-RUN CT_PROV_VERSION="v0.2.1"; \
-    wget https://github.com/coreos/terraform-provider-ct/releases/download/${CT_PROV_VERSION}/terraform-provider-ct-${CT_PROV_VERSION}-linux-amd64.tar.gz && \
-    tar xf terraform-provider-ct-${CT_PROV_VERSION}-linux-amd64.tar.gz -C /bin --strip=1
+# We need to build ct_config terraform provider from sources, because of two things:
+# 1. it's still not avaialble in terraform repos [1].
+# 2. latest binary release v0.2.1 is too old and does not support passwd groups.
+#
+# Links:
+# 1 - https://github.com/coreos/terraform-provider-ct/issues/21
+
+# Install Go 1.9.
```

Comment:
why not 1.10?

Reply:
we can do 1.10 no strong feelings, it has just released. will fix that